src/components/home/sections/volunteer.scss

```diff
@@ -1,20 +1,26 @@
 $dark-purple: #412F8A;
 
-#volunteer {
+#volunteer { 
+  width: 100%; 
   @media only screen and (min-width: 1024px) {
     flex-direction: row;
-
+    float: left; 
+    padding: 1em;
     @media only screen and (min-width: 1366px) {
-      width: 80%;
-      margin: auto;
+      width: 40%;
+      margin: auto; 
+      padding-left: 20vh;
```

Comment:
This is causing the left padding which adds to things being pushed too far to the right.

Reply:
I thought I took care of the extra space in the padding?

Reply:
This is still causing the issue on my end. Removing it removes the extra padding

With the padding:
<img width="724" alt="Screen Shot 2020-11-14 at 4 16 08 PM" src="https://user-images.githubusercontent.com/14047327/99159783-b5ad9780-2694-11eb-9233-72613c45eeeb.png">

Without:
<img width="751" alt="Screen Shot 2020-11-14 at 4 16 30 PM" src="https://user-images.githubusercontent.com/14047327/99159787-c2ca8680-2694-11eb-8181-811a49407b1d.png">

Reply:
If you've removed this, can you double check that you pushed your local changes up. I'm still seeing this here on github
https://github.com/nourimeals/website/blob/fc2f95f0a4d878d8ff49d3dcc1f5f7070b8a19d9/src/components/home/sections/volunteer.scss#L12